docs/compatibility-table.mdx

```diff
@@ -4,21 +4,22 @@ title: Compatibility table
 
 # Compatibility table
 
-The following table describes which versions of `patrol`
-and `patrol_cli` are compatible with each other.
+The following table describes which versions of `patrol` , `patrol_cli` and `flutter` are compatible with each other and 
 The simplest way to ensure that both packages are compatible
 is by always using the latest version. However,
 if for some reason that isn't possible, you can refer to 
 the table below to assess which version you should use.
 
-|   patrol_cli   |     patrol     |
-| -------------- | -------------- |
-| 3.2.0 -        | 3.11.0 -       |
-| 3.1.0 - 3.1.1  |     3.10.0     |
-| 2.6.5 - 3.0.1  | 3.6.0 - 3.10.0 |
-| 2.6.0 - 2.6.4  | 3.4.0 - 3.5.2  |
-| 2.3.0 - 2.5.0  | 3.0.0 - 3.3.0  |
-| 2.2.0 - 2.2.2  | 2.3.0 - 2.3.2  |
-| 2.0.1 - 2.1.5  | 2.0.1 - 2.2.5  |
-| 2.0.0          | 2.0.0          |
-| 1.1.4 - 1.1.11 | 1.0.9 - 1.1.11 |
+| patrol_cli            | patrol            | min flutter version |
+|-----------------------|-------------------|---------------------|
+| 3.2.0+                | 3.11.0+           | 3.22+               |
+| 3.1.0 - 3.1.1         | 3.10.0            | 3.10                |
+| 2.6.5 - 3.0.1         | 3.6.0 - 3.10.0    | 3.0                 |
+| 2.6.0 - 2.6.4         | 3.4.0 - 3.5.2     | 2.10                |
+| 2.3.0 - 2.5.0         | 3.0.0 - 3.3.0     | 2.5                 |
+| 2.2.0 - 2.2.2         | 2.3.0 - 2.3.2     | 2.0                 |
+| 2.0.1 - 2.1.5         | 2.0.1 - 2.2.5     | 2.0                 |
+| 2.0.0                 | 2.0.0             | 2.0                 |
+| 1.1.4 - 1.1.11        | 1.0.9 - 1.1.11    | 1.x                 |
```

Comment:
This table is still not correct. i.e. for patrol `2.3.0` minimum Flutter version is `3.3.0`.